libs/sql-schema-describer/src/walkers.rs

```diff
@@ -119,6 +119,18 @@ impl<'a> ColumnWalker<'a> {
         self.column().auto_increment
     }
 
+    /// Is this column indexed?
+    pub fn is_covered_by_index(&self) -> bool {
+        let table = self.table();
+        let name = self.name();
+
+        table.indexes().any(|idx| idx.contains_column(name))
+            || table
+                .primary_key()
+                .map(|pk| pk.column_names().any(|c_name| c_name == name))
+                .unwrap_or(false)
```

Comment:
worried that "indexed" here is ambiguous — should we have is_covered_by_secondary_index and is_covered_by_pk?

Reply:
part_of or covered?

Reply:
yes.